ru-ru/linker-ru.html.markdown

```diff
@@ -0,0 +1,204 @@
+---
+category: tool
+tool: linker
+contributors:
+    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
+translators:
+    
```

Comment:
Please change this to

```
contributors:
    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
translators:
    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
```

and also remove the ```filename:``` line

Reply:
Thank you. I made changes which you asked.